common-template/src/main/resources/templates/slack/Default/beta/default.json

```diff
@@ -0,0 +1,40 @@
+{
+    "blocks": [
+        {
+            "type": "header",
+            "text": {
+                "type": "plain_text",
+                "text": "{data.source.application.display_name} - {data.source.bundle.display_name}",
+                "emoji": true
+            }
+        },
+        {#if data.severity.asSeverityEmoji??}
```

Comment:
I think we should test the property itself instead of the output of the extension.

Reply:
Same comment other Slack templates